src/components/shared/Entity/Header.tsx

```diff
@@ -8,65 +7,24 @@ import {
     Toolbar,
     Typography,
 } from '@mui/material';
-import { EditorStoreState } from 'components/editor/Store';
 import ValidationErrorSummary from 'components/shared/Entity/ValidationErrorSummary';
-import { DraftSpecQuery } from 'hooks/useDraftSpecs';
 import { useRouteStore } from 'hooks/useRouteStore';
-import { useZustandStore } from 'hooks/useZustand';
 import { ReactNode } from 'react';
-import { FormattedMessage } from 'react-intl';
-import {
-    entityCreateStoreSelectors,
-    formInProgress,
-    FormStatus,
-} from 'stores/Create';
+import { entityCreateStoreSelectors, formInProgress } from 'stores/Create';
 
 interface Props {
-    test: (event: any) => void;
-    testDisabled: boolean;
-    save: (event: any) => void;
-    saveDisabled: boolean;
-    formId: string;
+    TestButton: ReactNode;
+    SaveButton: ReactNode;
```

Comment:
Passing component instead of the previous settings allowing us to house logic in the components.